damnit/gui/main_window.py

```diff
@@ -592,17 +592,29 @@ def inspect_data(self, index):
             pp.show()
             return
 
+        if variable.type_hint() is DataType.Dataset:
+            QMessageBox.warning(self, "Can't inspect variable",
+                                f"'{quantity}' is a Xarray Dataset (not supported).")
+
         try:
-            data = xr.DataArray(variable.read())
+            data = variable.read()
         except KeyError:
             log.warning(f'"{quantity}" not found in {variable.file}...')
             return
 
+        if variable.type_hint() is DataType.DataArray:
```

Comment:
If DataArrays are always handled by this branch, I think we can simplify the code below a bit. It creates a DataArray object and checks for a trainId coordinate, which it will now never find. So we could probably just make that code work with a numpy array.